packages/api-client/src/views/Request/RequestSidebar.vue

```diff
@@ -164,7 +164,7 @@ onBeforeUnmount(() => {
             @openMenu="(item) => Object.assign(menuItem, item)">
             <template #leftIcon>
               <LibraryIcon
-                class="text-sidebar-c-2 size-3.5 group-hover:hidden"
+                class="text-sidebar-c-2 size-3.5 stroke-[2.5] group-hover:hidden"
```

Comment:
Is this supposed to be thicker than in the icon selector?

Reply:
used the same as the icon arrow here, but `2.5` in the selector felt a bit much for scanning, feel free to try, no strong opinion here!